src/main/java/com/sap/oss/phosphor/fosstars/model/score/oss/SecurityReviewScore.java

```diff
@@ -0,0 +1,62 @@
+package com.sap.oss.phosphor.fosstars.model.score.oss;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.SECURITY_REVIEWS;
+import static com.sap.oss.phosphor.fosstars.model.other.Utils.findValue;
+
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.score.FeatureBasedScore;
+import com.sap.oss.phosphor.fosstars.model.value.ScoreValue;
+import com.sap.oss.phosphor.fosstars.model.value.SecurityReview;
+import com.sap.oss.phosphor.fosstars.model.value.SecurityReviews;
+import java.time.Duration;
+import java.time.Instant;
+
+/**
+ * This scoring function checks if and how security reviews have been done
+ * for an open source project. The score is based on
+ * {@link com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures#SECURITY_REVIEWS} feature.
+ */
+public class SecurityReviewScore extends FeatureBasedScore {
+
+  /**
+   * Create a new scoring function.
+   */
+  public SecurityReviewScore() {
+    super("How security reviews have been done for an open source project", SECURITY_REVIEWS);
+  }
+
+  @Override
+  public ScoreValue calculate(Value<?>... values) {
+    Value<SecurityReviews> reviews = findValue(values, SECURITY_REVIEWS,
+        "Hey! You have to tell me about security reviews!");
+
+    ScoreValue score = scoreValue(MIN, reviews);
+    if (reviews.isUnknown()) {
+      return score.explain("No info about security reviews");
+    }
+
+    if (reviews.get().isEmpty()) {
+      return score.explain("No security reviews have been done");
+    }
+
+    double value = 0.0;
+    Instant now = Instant.now();
+    for (SecurityReview review : reviews.get()) {
+      value += pointsFor(review, now);
+    }
+
+    return score.set(value);
+  }
+
+  /**
+   * Calculate points for a security review.
+   *
+   * @param review The security review.
+   * @param now Current time.
+   * @return Points for the security review.
+   */
+  static double pointsFor(SecurityReview review, Instant now) {
+    long years = (Duration.between(review.date().toInstant(), now).toDays() / 365) + 1;
+    return MAX / years;
```

Comment:
As I understand here
- The max time duration is taken for a review. The lower the point is added to the score.

Reply:
Please let me know why the duration between the review date and now makes sense?

Reply:
The older review is, the more likely it is that new changes have been added. Those new changes were not covered by the review. Plus, new vulnerability types and attacks may be discovered since the review.

In general, it would be good to update the scoring function to take into account the changes in the project after review. The less changes, the less impact on the score. For example, we can use commit history.

Reply:
Sounds good. Do we open an issue for the same? Or would you want to work on this PR?

Reply:
I would prefer to open a separate issue for that.

Reply:
https://github.com/SAP/fosstars-rating-core/issues/586